.rubocop_todo.yml

```
@@ -59,6 +66,64 @@ Metrics/ParameterLists:
   Exclude:
     - 'dist/clouduploader.rb'
 
+# Offense count: 6
+RSpec/BeforeAfterAll:
+  Exclude:
+    - 'spec/spec_helper.rb'
+    - 'spec/rails_helper.rb'
+    - 'spec/support/**/*.rb'
```

```suggestion
    - 'spec/support/capybara.rb'
```
There is only one file, inside `dist/t/spec/support/`. I guess if you modified this `.rubocop_todo.yml` manually, or was automatically generated...

---

Same here...